.pre-commit-config.yaml

```diff
@@ -0,0 +1,62 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+default_language_version:
+  python: python3.10
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
```

Comment:
let's do [pre-commit autoupdate](https://pre-commit.com/#pre-commit-autoupdate)  
don't forget to check versions here and in pyproject.toml are matching (e.g. for black)